src/html/markup.mli

```diff
@@ -19,19 +19,36 @@
 module Html = Tyxml.Html
 
 
-
 val keyword : string -> [> Html_types.span ] Html.elt
 
-module Type : sig
-  val path :
-    [< Html_types.span_content_fun ] Html.elt list ->
+module ML : sig
+  module Type : sig
+    val path :
+      [< Html_types.span_content_fun ] Html.elt list ->
       [> Html_types.span ] Html.elt
-  val var : string -> [> Html_types.span ] Html.elt
+    val var : string -> [> Html_types.span ] Html.elt
+  end
+
+  val arrow : [> Html_types.span ] Html.elt
+  (** "->" with a non breaking hyphen, styled as a keyword. *)
+
+  val label : Model.Lang.TypeExpr.label -> [> `PCDATA ] Html.elt list
+  (** For optional arguments adds a word joiner between the question mark and the
+      label. *)
 end
 
-val arrow : [> Html_types.span ] Html.elt
-(** "->" with a non breaking hyphen, styled as a keyword. *)
+module RE : sig
```

Comment:
It's probably worth introducing a common module interface to make the signature more DRY